cms/guidelines/an-open-invitation-to-developers.md

```diff
@@ -5,4 +5,4 @@ date: 2020-01-06T16:00:00.000Z
 
 The Ethereum main network is currently the canonical blockchain for our reference implementation at our website - [tradetrust.io](https://tradetrust.io) . Ethereum currently remains our choice of a data backend as it allows for permissionless onboarding, and minimises any quarrels over ownership of the infrastructure. Ethereum 2.0 offers a promising solution that is scalable and addresses some environmental concerns, but it is currently not yet ready.
 
-Moving forward, other suitable blockchains that have the mandatory properties and features may be considered to be offered as alternative options for the TradeTrust framework. The conditions will serve as a guideline to identify suitable candidates for consideration. We’d like to invite you to work with us to expand the list of blockchain options for TradeTrust. Feel free to contact us to discuss further.
+Moving forward, other suitable blockchains that have the mandatory properties and features may be considered to be offered as alternative options for the TradeTrust framework. The conditions will serve as a guideline to identify suitable candidates for consideration. We’d like to invite you to work with us to expand the list of blockchain options for TradeTrust. Feel free to [contact us](/contact) to discuss further.
```

Comment:
um i don't see these markdown being used by the CMS currently

Reply:
ok added